param_persist/sqlalchemy/models/instance_model.py

```diff
@@ -0,0 +1,29 @@
+"""The instance model for the param sqlalchemy features.
+
+This file was generated on July 30, 2020
+"""
+__author__ = "Gage Larsen"
+__copyright__ = "Copyright: (c) Aquaveo 2020"
+__maintainer__ = "Gage Larsen"
+__email__ = "glarsen@aquaveo.com"
+
+import uuid
+
+from sqlalchemy import Column, String
+from sqlalchemy.orm import relationship
+
+from param_persist.sqlalchemy.models import Base
+
+
+class InstanceModel(Base):
+    """The InstanceModel."""
+    __tablename__ = 'instances'
+
+    id = Column(String, primary_key=True, default=str(uuid.uuid4()), unique=True, nullable=False)
```

Comment:
The String type maps to varchar in most databases I believe. It would probably be more efficient to use a Char field with set length since we are using UUID's that have a set length.